src/Testing/src/Microsoft.AspNetCore.Testing.csproj

```
@@ -45,6 +44,11 @@
     <Reference Include="xunit.extensibility.execution" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' OR
+      '$(MSBuildRestoreSessionId)' == '' ">
```

Letting current build run w/ 🤞 but this probably needs
```suggestion
  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' OR
      '$(TargetFramework)' == '$(DefaultNetFxTargetFramework)' OR
      '$(MSBuildRestoreSessionId)' == '' ">
```
Not sure why but #33560 already has this.

---

I see, I may have accidently removed it with a4d926140529839b489672ccc25d0a07d1c096e3.  I think I may have misread that condition as NetCore rather than NetFx.  Apologies for mucking this up.